kitchen-habitat.gemspec

```diff
@@ -4,8 +4,8 @@ require "kitchen-habitat/version"
 Gem::Specification.new do |s|
   s.name              = "kitchen-habitat"
   s.version           = Kitchen::Habitat::VERSION
-  s.authors           = ["Steven Murawski"]
-  s.email             = ["smurawski@chef.io"]
+  s.authors           = ["Steven Murawski", "Robb Kidd"]
+  s.email             = ["smurawski@chef.io", "robb@thekidds.org"]
```

Comment:
@smurawski, would you like to take this opportunity to update your email here?

Reply:
Done.  Thanks @robbkidd